string configName
string configObject
---
bool success